float64 heading
float64 timeLimit
float64 range
---
int8 approach
bool success